modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -96,10 +96,12 @@ ifdef::azure[]
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/04_bootstrap.json" \
   --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters baseName="${INFRA_ID}" \ <2>
+  --parameter bootstrapVMsize <3>
```

Comment:
Shouldn't this also have a value with it like how the others are formated? I guess like 
```suggestion
  --parameter bootstrapVMsize="${stuff}" <3>
```
I'm not sure though

Reply:
You're correct. I suggest `Standard_D4s_v3` which is the value used by the template when the `--parameter bootstrapVMSize` is not specified.